kolibri/content/utils/sqlalchemybridge.py

```diff
@@ -103,15 +98,26 @@ def set_all_class_defaults(Base):
             pass
 
 
-def get_base(connection_string, engine, app_name=None):
+SCHEMA_PATH_TEMPLATE = os.path.join(os.path.dirname(__file__), '../fixtures/{name}_content_schema')
+
+def prepare_bases():
+
+    for name in CONTENT_DB_SCHEMA_VERSIONS:
+
+        with open(SCHEMA_PATH_TEMPLATE.format(name=name), 'rb') as f:
+            metadata = pickle.load(f)
+        BASES[name] = prepare_base(metadata=metadata)
+
+
+def prepare_base(engine=None, metadata=None, app_name=None):
     """
     Get a Base mapping for a particular database engine and Django app
     A Base mapping defines the mapping from database tables to the SQLAlchemy ORM and is
     our main entrypoint for interacting with arbitrary databases without a predefined schema
     """
-    cache_key = '{connection}_{app_name}'.format(connection=connection_string, app_name=(app_name or 'all'))
-    if cache_key not in BASE_CLASSES_CACHE:
+    if not metadata:
```

Comment:
Leaves the reflection logic in here, just in case - but it's not used. Can remove if needed.

Reply:
Let's just remove it then -- dead code may turn into a zombie if left too long.